src/components/SEO.js

```diff
@@ -10,6 +10,22 @@ const METADATA = [
   },
 ];
 
+const crazyEgg = (location) => {
+  const homepage = 'https://docs.newrelic.com';
+  const signup =
+    'https://docs.newrelic.com/docs/accounts/accounts-billing/account-setup/create-your-new-relic-account';
+
+  if (location === homepage || signup) {
```

Comment:
`location` is a little different from [`window.location`](https://developer.mozilla.org/en-US/docs/Web/API/Window/location) - it's not a plain string, but rather an object (you can see that in the prop-types below). Our [documentation](https://github.com/newrelic/gatsby-theme-newrelic/tree/develop/packages/gatsby-theme-newrelic#seo) on the `SEO` component doesn't specifiy the _exact_ shape of that object, but it does say that it should have a `pathname` field that we can use for this purpose.

Additionally, javascriprt will treat the `|| signup` as a separate condition on it's own and not attempt to verify if `signup` is equal to `location`. You need to supply `location ===` for both statements.

Here's the adjustments to the code that should handle both those things:

```suggestion
  const { pathname } = location;
  
  const homepage = '/';
  const signup = '/docs/accounts/accounts-billing/account-setup/create-your-new-relic-account/';

  if (pathname === homepage || pathname === signup) {
```

Reply:
Thank you so much, @zstix! I wasn't sure about the location prop and whether it should be a relative path or an external link. I'll go ahead and merge your suggestion 👏🏻👏🏻👏🏻